packages/@okta/vuepress-site/guides/sign-into-spa/configure-the-sdk/index.md

```diff
@@ -14,9 +14,14 @@ const config = {
   clientId: '{clientId}',
   issuer: 'https://{yourOktaDomain}.com/oauth2/default',
```

Comment:
@vijetmahabaleshwar-okta Can you modify this to `https://{yourOktaDomain}/oauth2/default` (drop the `.com`)